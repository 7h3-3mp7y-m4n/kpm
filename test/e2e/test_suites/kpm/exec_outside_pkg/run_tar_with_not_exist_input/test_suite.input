kpm run --input no_exist --vendor kcl1.tar