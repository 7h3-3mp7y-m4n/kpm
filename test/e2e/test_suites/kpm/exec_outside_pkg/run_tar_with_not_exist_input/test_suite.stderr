kpm: entry file cannot be found, please make sure the '--input' entry file can be found
